fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,204 @@
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CSelect
+                        label={"Land"}
+                        defaultValue={[]}
+                        value={country}
+                        onChange={(event) => setCountry(event.target.value)}
+                        options={countries}
+                        validators={["required"]}
+                        errorMessages={["Die Auswahl eines Landes ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CSelect
+                        label={"Anrede"}
+                        defaultValue={[]}
+                        value={gender}
+                        onChange={(event) => setGender(event.target.value)}
+                        options={genderMenuItems}
+                        validators={["required"]}
+                        errorMessages={["Die Auswahl einer Anrede ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CTextField
+                        label={"Vorname"}
+                        value={firstname}
+                        onChange={(event) => setFirstname(event.target.value)}
+                        validators={["required"]}
+                        errorMessages={["Die Eingabe eines Vornamen ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={4}>
+                    <CTextField
+                        label={"Nachname"}
+                        value={lastname}
+                        onChange={(event) => setLastname(event.target.value)}
+                        validators={["required"]}
+                        errorMessages={["Die Eingabe eines Nachnamen ist erforderlich!"]}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <CTextField
+                        label={"E-Mail"}
+                        value={mail}
+                        onChange={(event) => setMail(event.target.value)}
+                        validators={["required", "isEmail"]}
+                        errorMessages={["Die Eingabe einer E-Mail-Adresse ist erforderlich!", "Die Eingegebene E-Mail-Adresse ist nicht valide!"]}
+                    />
+                </Grid>
+                <Grid item md={6}>
+                    <FormControlLabel
+                        control={<Checkbox
+                            className={classes.checkbox}
+                            checked={sendConfirmationMail}
+                            onChange={() => setSendConformationMail(!sendConfirmationMail)}
+                            color="primary"
+                        />}
+                        label="Bestätigungsmail versenden?"
+                    />
+                </Grid>
+            </Grid>
+            <Stack spacing={2} direction="row" className={classes.footer}>
```

Comment:
Codacy has a fix for the issue: [Replace `············` with `········`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
        <Stack spacing={2} direction="row" className={classes.footer}>
```